Dockerfile

```
@@ -1,15 +1,12 @@
 FROM node:23.10.0-alpine3.21
 
-RUN apk add --no-cache python3=3.12.9-r0 make=4.4.1-r2 g++=14.2.0-r4 \
+RUN apk add --no-cache python3=3.12.10-r0 make=4.4.1-r2 g++=14.2.0-r4 curl \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)